lldb/include/lldb/Host/aix/Ptrace.h

```diff
@@ -0,0 +1,44 @@
+//===-- Ptrace.h ------------------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// This file defines ptrace functions & structures
+
+#ifndef liblldb_Host_aix_Ptrace_h_
```

Comment:
These should BE_ALL_UPPER_CASE. I know it's like that in the Linux one, but that's a mistake.

Reply:
This is the old include guard style. I guess Jonas only updated the files that are built on mac when he was doing the conversion. New files should definitely use the new style though..

Reply:
Okay, Done!